src/main/webapp/resources/js/components/ncbi/details/NcbiExportDetailsView.tsx

```diff
@@ -118,10 +122,10 @@ function NcbiExportDetailsView(): JSX.Element {
                 />
               )}
 
-              {bioSampleFile.files.singles.length > 0 && (
+              {bioSample.files.singles.length > 0 && (
```

Comment:
Change this to `bioSample.files.singles?.length`

Reply:
In this case, it is actually not necessary.  TypeScript enforces that all items of type `BioSampleDetails` have an attribute of `files` that has two attributes `singles` and `pairs` that are both arrays:
```typescript
export interface BioSampleDetails {
  key: string;
  details: BasicListItem[];
  files: {
    singles: SingleEndSequenceFile[];
    pairs: PairedEndSequenceFile[];
  };
}
```

Reply:
Perfect. Thanks!